modules/trusted-firmware-m/Kconfig.tfm

```diff
@@ -206,6 +206,20 @@ config TFM_BL2
 	  TFM is designed to run with MCUboot in a certain configuration.
 	  This config adds MCUboot to the build - built via TFM's build system.
 
+config TFM_BUILD_NS
+	bool "Build the TF-M Non-Secure application and libraries"
+	help
+	  When this is enabled, the TF-M build system will build the Non-Secure
+	  application and libraries. as a whole will be
```

Comment:
Nit: Capital A at the start of the sentence "**A**s a whole **(it)** will"
But maybe this sentence can be omitted completely I think.

Reply:
You're right. That is actually just a mistake.